src/Tizen.NUI.Components/Controls/RecyclerView/CollectionView.cs

```diff
@@ -25,14 +25,15 @@
 namespace Tizen.NUI.Components
 {
     /// <summary>
-    /// This class provides a View that can layouting items in list and grid with high performance.
+    /// Selectable RecyclerView that presenting a collection of items with variable layouters.
     /// </summary>
-    [EditorBrowsable(EditorBrowsableState.Never)]
+    /// <since_tizen> 9 </since_tizen>
     public class CollectionView : RecyclerView
     {
         /// <summary>
         /// Binding Property of selected item in single selection.
         /// </summary>
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
If you want to open the property, please remove [EditorBrowsable(EditorBrowsableState.Never)].